main.js

```diff
@@ -14,11 +14,38 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
 async function acceptStudent(id, fullName, age, address){
     let attributes = { ID:id, Name: fullName, Age: age, Address: address };
     await db.put(id, attributes);
-    return await db.get(id, function(err, value){
+    await db.get(id, function(err, value){
         if(err){
             console.log(err);
         } else{
             console.log(value, status[0]);
         }
     })
+    var scheduleDate = new Date('March 1, 2021 08:30:00');
+    return await scheduleInterview(id, scheduleDate);
+}
+
+async function scheduleInterview(id, scheduleDate){
+    await db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, status[1]);
+            console.log('Interview date on', scheduleDate);
+        }
+    })
+    return await scheduleExam(id, scheduleDate);
 }
+
+async function scheduleExam(id, scheduleDate){
+    var examDate = new Date(scheduleDate);
+    examDate.setDate(examDate.getDate() + 10);
```

Comment:
why + 10, what's the reason

Reply:
sir, it is just a example....assuming that 10 days after the interview, the student is set to have the exam. Anyways, it is automated and it can still be changed.

Reply:
we also need to set examDate and scheduleDate differently